xbmc/windowing/gbm/DRMUtils.cpp

```diff
@@ -558,6 +558,12 @@ bool CDRMUtils::OpenDrm(bool needConnector)
 
       CLog::Log(LOGDEBUG, "CDRMUtils::%s - opened device: %s using module: %s", __FUNCTION__, drmGetDeviceNameFromFd2(m_fd), module);
 
+      m_renderFd.attach(drmOpenWithType(module, nullptr, DRM_NODE_RENDER));
+      if (m_fd)
```

Comment:
Guess you want to check m_renderFd here?

Reply:
whoops